configure.ac

```diff
@@ -227,6 +228,15 @@ AS_IF([test "x$with_docs_only" == xyes],
      #
      AC_SUBST([VALGRIND_LIBPATH], [${valgrind_libpath}])
 
+     #
+     # Enable examples build
+     #
+     AC_ARG_ENABLE([examples],
+                   [AC_HELP_STRING([--enable-examples],
+                                   [Enable examples build])],
+                   [AM_CONDITIONAL([HAVE_EXAMPLES], [true])],
```

Comment:
according to this, doing --disable-examples will also enable examples. need to check value of $enable_examples
see https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Package-Options.html